src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-count/jj-count.component.ts

```diff
@@ -326,6 +344,22 @@ export class JJCountComponent implements OnInit {
       this.form.get('revisedDueDate').setValue(this.jjDisputedCount?.dueDate);
     }
   }
+
+  //Latest Plea
+  bindLatestPlea(value){
+    this.form.controls.latestPlea.setValue(value);
+  }
+
+  bindLatestPleaUpdateTs(value){
+    this.form.controls.latestPleaUpdateTs.setValue(this.toDateString(new Date(value)));
+  }
+
+  toDateString(date: Date): string {
```

Comment:
Can we add this to DateUtil?
/src/app/shared/utils/date-util.ts

Reply:
I think we can store it as static variable

Reply:
Please use datepipe to handle it, I have implemented similar coding in other components, you can reference to it